app/utils/notification_sender.py

```
@@ -0,0 +1,24 @@
+    payload = {
+        "title": title,
+        "message": message,
+        "type": notification_type
+    }
+
+    try:
+        response = requests.post(url, json=payload, timeout=5)
+        response.raise_for_status()
+        return response.json()
+    except requests.RequestException:
+        raise APIException(ErrorStatus.NOTIFICATION_SEND_FAILED)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)